public/styles/common.css

```diff
@@ -276,6 +276,30 @@ button {
 	}
 }
 
+img.packageLogo {
+	float: right;
+	max-width: 150px;
+	max-height: 150px;
+	margin: 0 0 1em 1em;
+}
+
+@media only screen and (max-width: 480px) {
+	img.packageLogo {
+		display: block;
+		float: none;
+		max-width: 100%;
```

Comment:
made it so it isn't higher than 220px now, it can still be a wide logo but square logo will be smaller now. It does not stretch (at least not in chrome)

Reply:
Shouldn't we require logos to be square and convert them otherwise? This usually makes such design things a lot easier.

Reply:
hm I'm actually a fan of supporting flat logos such as banners, isn't that much more work with html and css